src/backend/distributed/commands/utility_hook.c

```diff
@@ -1641,3 +1674,46 @@ RunPostprocessMainDBCommand(Node *parsetree)
 		RunCitusMainDBQuery(mainDBQuery->data);
 	}
 }
+
+
+/*
+ * IsStatementSupportedIn2Pc returns true if the statement is supported in 2pc
```

Comment:
```suggestion
 * IsStatementSupportedIn2Pc returns true if the statement is supported from a
 * non-main database.
```

Also, maybe rename the function to IsStatementSupportedFromNonMainDbs?